db/version_set_test.cc

```diff
@@ -1196,10 +1196,8 @@ TEST_F(VersionSetTest, WalEditsNotAppliedToVersion) {
     edits.back()->AddWal(i, WalMetadata(i));
   }
   // Delete the first half of the WALs.
-  for (uint64_t i = 1; i <= kNumWals; i++) {
-    edits.emplace_back(new VersionEdit);
-    edits.back()->DeleteWal(i);
-  }
+  edits.emplace_back(new VersionEdit);
```

Comment:
Seems like we're leaking this edit.

Reply:
`autovector<std::unique_ptr<VersionEdit>> edits;`, unique_ptr should `delete` them on destruction.

Reply:
Thanks, I missed that.